dash/development/base_component.py

```diff
@@ -2,7 +2,19 @@
 import copy
 import os
 import inspect
+import json
 import keyword
+import cerberus
+
+
+
+# Forward declated Component class, see below.
```

Comment:
did you mean 'declared'? PEP8 requires one space less :)

PS : this is my first review on dash code and I am mainly reading the code you wrote to learn more, my suggestion will mostly be about styling and spelling, I apologize in advance for that!

Reply:
> my suggestion will mostly be about styling and spelling, I apologize in advance for that!

Note that we enforce PEP8 in our tests here: https://github.com/plotly/dash/blob/4d93652e40b55eb94c75359f5d47013051839ecf/.circleci/config.yml#L38-L39

So, it's not necessary to review PEP8 issues in our PRs as they will get fixed before it gets merged into master.